File: Java/Recursion/Fibonati.java
package Java.Recursion;

public class Fibonati {
    public int fib(int n) {
        if (n == 0) {
            return 0;
        }
        if (n == 1) {
            return 1;
        }
        return fib(n - 1) + fib(n - 2);
    }

    public static void main(String[] args) {
        Fibonati f = new Fibonati();
        for (int i = 0; i < 20; i++) {
            System.out.print(f.fib(i) + " ,");
        }
    }
}

File: Python/Recursion/Fibonati.py
class Fibonati(object):
    def FibonatiClass(self, n):
        if n == 0:
            return 0
        if n == 1:
            return 1
        return self.FibonatiClass(n - 1) + self.FibonatiClass(n - 2)
    
Fibonati = Fibonati()

print(Fibonati.FibonatiClass(6))